BuildResidentialHPXML/measure.rb

```diff
@@ -0,0 +1,3573 @@
+    arg.setDisplayName("Heat Pump: Cooling Capacity")
+    arg.setDescription("The output cooling capacity of the heat pump. If using '#{Constants.SizingAuto}', the autosizing algorithm will use ACCA Manual S to set the capacity.")
+    arg.setUnits("Btu/hr")
+    arg.setDefaultValue(Constants.SizingAuto)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument("heat_pump_fraction_heat_load_served", true)
+    arg.setDisplayName("Heat Pump: Fraction Heat Load Served")
+    arg.setDescription("The heat load served fraction of the heat pump.")
+    arg.setUnits("Frac")
+    arg.setDefaultValue(1)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument("heat_pump_fraction_cool_load_served", true)
+    arg.setDisplayName("Heat Pump: Fraction Cool Load Served")
+    arg.setDescription("The cool load served fraction of the heat pump.")
+    arg.setUnits("Frac")
+    arg.setDefaultValue(1)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument("heat_pump_backup_fuel", heat_pump_backup_fuel_choices, true)
+    arg.setDisplayName("Heat Pump: Backup Fuel Type")
+    arg.setDescription("The backup fuel type of the heat pump.")
+    arg.setDefaultValue("none")
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument("heat_pump_backup_heating_efficiency", true)
+    arg.setDisplayName("Heat Pump: Backup Rated Efficiency")
+    arg.setDescription("The backup rated efficiency value of the heat pump. Percent for electricity fuel type. AFUE otherwise.")
+    arg.setDefaultValue(1)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeStringArgument("heat_pump_backup_heating_capacity", true)
+    arg.setDisplayName("Heat Pump: Backup Heating Capacity")
+    arg.setDescription("The backup output heating capacity of the heat pump. If using '#{Constants.SizingAuto}', the autosizing algorithm will use ACCA Manual S to set the capacity.")
+    arg.setUnits("Btu/hr")
+    arg.setDefaultValue(Constants.SizingAuto)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument("mini_split_is_ducted", true)
+    arg.setDisplayName("Mini-Split: Is Ducted")
+    arg.setDescription("Whether the mini-split heat pump is ducted or not.")
+    arg.setDefaultValue(false)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument("evap_cooler_is_ducted", true)
+    arg.setDisplayName("Evaporative Cooler: Is Ducted")
+    arg.setDescription("Whether the evaporative cooler is ducted or not.")
+    arg.setDefaultValue(false)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument("heating_system_flow_rate", true)
+    arg.setDisplayName("Heating System: Flow Rate")
+    arg.setDescription("The flow rate of the heating system.")
+    arg.setUnits("CFM")
+    arg.setDefaultValue(0)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument("cooling_system_flow_rate", true)
```

Comment:
Remove these `heating_system_flow_rate` and `cooling_system_flow_rate` arguments.

Reply:
So just enter `nil` for `heating_cfm` and `cooling_cfm`?

Reply:
They already default to nil, don't need to even pass them to the HPXML.add methods.